binder/jupyter_notebook_config.py

```diff
@@ -0,0 +1,30 @@
+lab_command = ' '.join([
+    'jupyter',
+    'lab',
+    '--dev-mode',
+    '--debug',
+    '--no-browser',
+    '--port={port}',
+    '--NotebookApp.token=""',
+    '--NotebookApp.base_url={base_url}lab-dev',
+    # Disable dns rebinding protection here, since our 'Host' header
+    # is not going to be localhost when coming from hub.mybinder.org
+    '--NotebookApp.allow_remote_access=True'
+])
+
+c.ServerProxy.servers = {
+    'lab-dev': {
+        'command': [
+            '/bin/bash', '-c',
+            # Redirect all logs to a log file
+            f'{lab_command} >jupyterlab-dev.log 2>&1'
```

Comment:
@bollwyvl The logs _are_ going to a file 👍

Reply:
(ooh, and maybe a pre-configued `/lab-dev/logs/` workspace #5903 with said logs open... :cake: )

Reply:
I like the idea of the _logs! We can add more after too.

Reply:
I like this idea as well. The `lab-dev` logs workspace is very completist :smile: